options/locale/locale_en-US.ini

```
@@ -2446,6 +2446,13 @@ settings.thread_id = Thread ID
 settings.matrix.homeserver_url = Homeserver URL
 settings.matrix.room_id = Room ID
 settings.matrix.message_type = Message Type
+settings.visibility.private.button = Make Private
+settings.visibility.private.text = Making private the repo will do that only allowed members can see.
+settings.visibility.public.button = Make Public
+settings.visibility.public.text = Making public the repo will do that anyone can see.
```

Similar to my last comment:
```suggestion
settings.visibility.public.text = Changing the visibility to public will make the repo visible to anyone.
```